packages/runtimes/js/src/JSRuntime.js

```diff
@@ -87,7 +87,7 @@ export default new Runtime({
           assets.push({
             filePath: __filename,
             code: `module.exports = ${JSON.stringify(
-              dependency.moduleSpecifier,
+              dependency.moduleSpecifier, // TODO remove pipeline
```

Comment:
@wbinnssmith ?

Reply:
In this case, we couldn't resolve the url internally, so it's nothing Parcel handled in the first place. Was this reached?

Reply:
> we couldn't resolve the url internally

You mean we ignored it?

No, this wasn't a problem in this case. But I looked through all references to `moduleSpecifier` where it could be used (or was used - see my comment above) as an URL.